design-tokens/force-base/border-color.yml

```diff
@@ -11,28 +11,28 @@ imports:
 - ./aliases/color.yml
 props:
   COLOR_BORDER:
-    value: '{!GEYSER}'
+    value: '{!WARM-GREY-2}'
     cssProperties:
     - 'border*'
     - box-shadow
     - 'background*'
     comment: Default border color for UI elements.
   COLOR_BORDER_BRAND:
-    value: '{!AZURE}'
+    value: '{!PICSA_GREEN-1}'
     cssProperties:
     - 'border*'
     - box-shadow
     - outline
     - outline-color
     comment: Our product brand blue.
   COLOR_BORDER_BRAND_DARK:
-    value: '{!SCIENCE_BLUE}'
+    value: '{!PICSA_GREEN-1}'
     comment: 'Our product brand blue, darkened to meet accessibility color contrast ratios with white text.'
```

Comment:
@NicolaasDieBaas I've made this change. Will commit shortly with some of the others.